src/subtitles-octopus.js

```diff
@@ -404,14 +407,38 @@ var SubtitlesOctopus = function (options) {
         }
     };
 
+    function _computeCanvasSize(width, height) {
+        var scalefactor = self.prescaleFactor <= 0 ? 1.0 : self.prescaleFactor;
+
+        if (height == 0) {
```

Comment:
I think it's better to check the width here as well

`if (height == 0 || width == 0) {`

Reply:
It's not strictly required for `_computeCanvas` to work, but sure, normalising all zero and negative cases to `(0, 0))`sounds good.
Now using `if (height <= 0 || width <= 0) {`.